NEWS.md

```diff
@@ -1,5 +1,10 @@
 # lintr (development version)
 
+## Breaking changes
+
+* Updated the location priority order that `lintr` checks to find a `.lintr` settings file. It now checks in the `.github/linters` project sub folder after the currently searched directory. This is to improve 
```

Comment:
I would start with ".lintr configs can now be kept in directory .github/linters' for better compatibility with Super-Linter. Note that this may be a breaking change if you already have a config in both such a directory and in your R project's root since the former will now be discovered first where it was ignored before." and also adding something like "Please see vignette(...) for details on how configs are discovered."

OTOH @AshesITR it seems pretty unlikely anyone's affected by this (why have a .lintr file in that particular place if it's not actually doing anything?), am I missing something?

Reply:
It's more like if you want super-linter to be less critical than your personal `~/.lintr`, e.g. for performance reasons.
I agree this is very unlikely to occur. Technically, it's still a breaking change so might be best to alert users.

Reply:
OK, makes sense, in that case we should mention how to accomplish moving your preferred config to the front on an ad hoc basis (e.g. set that config as env). unless we already do so in the vignette, in which case we're covered by "see vignette for details"

Reply:
That's already documented in the vignette, but as a workaround I'd recommend copying `~/.lintr` to `./.lintr` instead.

Reply:
Updated file as per comments